src/test/regress/expected/failure_create_database.out

```diff
@@ -0,0 +1,310 @@
+SET citus.enable_create_database_propagation TO ON;
+SET client_min_messages TO WARNING;
+CREATE FUNCTION get_temp_databases_on_nodes()
+RETURNS TEXT AS $func$
+  SELECT array_agg(DISTINCT result ORDER BY result) AS temp_databases_on_nodes FROM run_command_on_all_nodes($$SELECT datname FROM pg_database WHERE datname LIKE 'citus_temp_database_%'$$) WHERE result != '';
+$func$
+LANGUAGE sql;
+CREATE FUNCTION ensure_no_temp_databases_on_any_nodes()
+RETURNS BOOLEAN AS $func$
+  SELECT bool_and(result::boolean) AS no_temp_databases_on_any_nodes FROM run_command_on_all_nodes($$SELECT COUNT(*)=0 FROM pg_database WHERE datname LIKE 'citus_temp_database_%'$$);
+$func$
+LANGUAGE sql;
+-- cleanup any orphaned resources from previous runs
+CALL citus_cleanup_orphaned_resources();
+SET citus.next_operation_id TO 4000;
+ALTER SYSTEM SET citus.defer_shard_delete_interval TO -1;
+SELECT pg_reload_conf();
+ pg_reload_conf
+---------------------------------------------------------------------
+ t
+(1 row)
+
+SELECT pg_sleep(0.1);
+ pg_sleep
+---------------------------------------------------------------------
+
+(1 row)
+
+SELECT citus.mitmproxy('conn.kill()');
+ mitmproxy
+---------------------------------------------------------------------
+
+(1 row)
+
+CREATE DATABASE db1;
+ERROR:  connection to the remote node postgres@localhost:xxxxx failed with the following error: connection not open
+SELECT citus.mitmproxy('conn.allow()');
+ mitmproxy
+---------------------------------------------------------------------
+
+(1 row)
+
+SELECT get_temp_databases_on_nodes();
+ get_temp_databases_on_nodes
+---------------------------------------------------------------------
+
+(1 row)
+
+CALL citus_cleanup_orphaned_resources();
+SELECT ensure_no_temp_databases_on_any_nodes();
+ ensure_no_temp_databases_on_any_nodes
+---------------------------------------------------------------------
+ t
+(1 row)
+
+SELECT * FROM public.check_database_on_all_nodes($$db1$$) ORDER BY node_type, result;
+      node_type       |                                                              result
+---------------------------------------------------------------------
+ worker node (remote) | {"database_properties": null, "pg_dist_object_record_for_db_exists": false, "stale_pg_dist_object_record_for_a_db_exists": false}
+ worker node (remote) | {"database_properties": null, "pg_dist_object_record_for_db_exists": false, "stale_pg_dist_object_record_for_a_db_exists": false}
+(2 rows)
+
+SELECT citus.mitmproxy('conn.onQuery(query="^CREATE DATABASE").cancel(' || pg_backend_pid() || ')');
+ mitmproxy
+---------------------------------------------------------------------
+
+(1 row)
+
+CREATE DATABASE db1;
+WARNING:  Commands that are not transaction-safe may result in partial failure, potentially leading to an inconsistent state.
+If the problematic command is a CREATE operation, consider using the 'IF EXISTS' syntax to drop the object,
+if applicable, and then re-attempt the original command.
```

Comment:
We should remove this warning now.

Reply:
Opened an issue for other CREATE commands: https://github.com/citusdata/citus/issues/7527.

Will remove the warning for CREATE DATABASE soon.